ush/python/pygw/src/pygw/task.py

```diff
@@ -1,3 +1,4 @@
+import datetime as dt
```

Comment:
Can you use this [task.py](https://github.com/aerorahul/global-workflow/blob/feature/pyforecast/ush/python/pygw/src/pygw/task.py)

Reply:
If you want, I can open a PR into your branch.  I don't know how to test though.

Reply:
yes please do and then I can test/debug this afternoon or tomorrow